third_party/llvm-project/DWARFVisitor.cpp

```diff
@@ -49,19 +51,27 @@ template <typename T> void DWARFYAML::VisitorImpl<T>::traverseDebugInfo() {
   // TODO: This code appears to assume that abbreviation codes increment by 1
   // so that lookups are linear. In LLVM output that is true, but it might not
   // be in general.
+  // Create a map of [byte offset into the abbreviation section] => [index in
```

Comment:
how many map entries do we end up with here? I'm a little unclear on where these keys come from.

Reply:
One key per CU.

It is a little confusing because LLVM puts the CU's abbrevs in a DeclSet, and all those sets are just flattened out into AbbrevDecls. I pushed a detailed comment now.